web/app/js/components/Deployment.jsx

```diff
@@ -201,6 +207,24 @@ export default class Deployment extends React.Component {
     );
   }
 
+  renderPaths() {
+    return _.size(this.state.pathMetrics) === 0 ? null :
+      <div>
+        <div className="border-container border-neutral subsection-header">
+          <div className="border-container-content subsection-header">
+              Paths
+          </div>
+        </div>
+        <TabbedMetricsTable
+          resource="path"
+          metrics={this.state.pathMetrics}
+          hideSparklines={true}
+          lastUpdated={this.props.lastUpdated}
+          metricsWindow={this.props.metricsWindow}
+          pathPrefix={this.props.pathPrefix} />
```

Comment:
I think this component needs a `key` property. On the deployment detail page I see:

![image](https://user-images.githubusercontent.com/9226/34854691-06572336-f6f0-11e7-80bf-124c483f2865.png)

Reply:
oops, thanks for catching!